.github/workflows/test.yml

```diff
@@ -83,7 +83,9 @@ jobs:
           poetry install --verbose --no-root --sync
 
       - name: Build docker image from source
-        run: docker build --tag phylum-ci:from-src --cache-from phylumio/phylum-ci:latest .
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: docker build --tag phylum-ci:from-src --build-arg GITHUB_TOKEN --cache-from phylumio/phylum-ci:latest .
```

Comment:
It seems weird to add this in two different ways in the two different steps... This one sets the environment with `env`, but the other one (line 108) just passes the value directly on the command line... I don't have a preference between the two, but it seems like we should pick one and stick with it.

Reply:
There were two thoughts here:

* Save 3 lines of code since adding the build argument would cause the line to go past 120 chars (the soft limit in my IDE) and require breaking it up
* Show that there are two ways to specify the build argument...kind of like a reminder or self documentation
  * The alternate style (a "naked" build arg that has the same name as an existing envvar) is already documented in the `Dockerfile` comments...so this is kind of already covered

The point is taken in that it can be confusing as to why there are two different methods. A change was made to make them the same.